src/bin/server.cpp

```diff
@@ -27,6 +39,43 @@ int main(int argc, char* argv[]) {
     return 0;
   }
 
+  /**
+    * The following code handles the parameter `benchmark_data` which allows users to generate benchmark data when
+    * starting the hyrise server. This is not an ideal solution, but due to several users requests and our goal to
+    * facilitate easy evaluation of hyrise, we decided to integrate the data generation in to the server nonetheless.
+    */
+  if (parsed_options.count("benchmark_data")) {
+    auto benchmark_data_arg = parsed_options["benchmark_data"].as<std::string>();
+    std::vector<std::string> bechmark_data_config;
+
+    boost::trim_if(benchmark_data_arg, boost::is_any_of(":"));
+    boost::replace_all(benchmark_data_arg, "-", "");
+    boost::to_lower(benchmark_data_arg);
+    boost::split(bechmark_data_config, benchmark_data_arg, boost::is_any_of(":"), boost::token_compress_on);
```

Comment:
Could you give one-line comments (or just examples) on what these transformations do?

Reply:
Added.